cppwinrt/code_writers.h

```diff
@@ -2292,9 +2296,9 @@ struct WINRT_IMPL_EMPTY_BASES produce_dispatch_to_overridable<T, D, %>
                 continue;
             }
 
-            for (auto&& interface_name : interfaces)
+            for (auto&& [interface_name, info] : interfaces)
             {
-                w.write("        using impl::consume_t<D, %>::%;\n",
+                w.write(info.overridable ? "        using %T<D>::%;\n" : "        using impl::consume_t<D, %>::%;\n",
```

Comment:
[nitpick] It would be _slightly_ easier to read/compare if the two strings were indented the same on separate lines.  The leading spaces spreads out the content.

Reply:
Agreed. Updated.